src/components/Dashboard/dashboard-layout.tsx

```diff
@@ -14,8 +14,8 @@ export const DashboardLayout = ({ remainingUrls, children }: DashboardLayoutProp
                 <div className="flex items-center pb-8 w-full justify-between">
                     <h1 className="text-3xl md:text-4xl xl:text-4xl text-center text-white font-semibold">Your URLs</h1>
 
-                    {remainingUrls && (
-                        <h4>
+                    {remainingUrls !== undefined && (
```

Comment:
```suggestion
                    {!!remainingUrls && (
```
this will also check for null and 0 value